include/illixr/switchboard.hpp

```diff
@@ -23,6 +23,46 @@ namespace ILLIXR {
 
 using plugin_id_t = std::size_t;
 
+const std::vector<std::string> ignore_vars = {
+    "plugins",         "data", "demo_data", "install_prefix", "enable_offload", "enable_alignment", "enable_verbose_errors",
+    "enable_pre_sleep"};
+const std::vector<std::string> ENV_VARS = {"DEBUGVIEW_LOG_LEVEL",
```

Comment:
In the future (not in this PR is fine), it'd be nice to have each plugin declare their own env vars locally, instead of including it here.

Reply:
That could be tricky as we don't want the env vars list to depend on what plugins are enabled. As it stands the system will dynamically add an env var if an unknown one is requested. So perhaps we should only include the "system wide" env vars (debug level, etc) in the header and let any others be dynamically generated and stored.

Reply:
I trimmed down the list to be just those used by the runtime, since anything can be dynamically added.